go/vt/tabletserver/tx_pool.go

```diff
@@ -72,7 +72,7 @@ func NewTxPool(
 		timeout:     sync2.AtomicDuration(timeout),
 		poolTimeout: sync2.AtomicDuration(poolTimeout),
 		ticks:       timer.NewTimer(timeout / 10),
-		txStats:     stats.NewTimings("Transactions"),
+		txStats:     stats.NewTimings(name + "Transactions"),
```

Comment:
This may break the monitoring. I've reverted the change of txStats name and will use a single txPool object in the unit test.